cmd/virtual-worker/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.11
```

Comment:
Every other Dockerfile is at 3.15, is 3.11 necessary?

Reply:
ah no, I just created this a long time ago, I'll update this